pkg/cim/volume.go

```diff
@@ -0,0 +1,297 @@
+package cim
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/microsoft/wmi/pkg/base/query"
+	"github.com/microsoft/wmi/pkg/errors"
+	cim "github.com/microsoft/wmi/pkg/wmiinstance"
+	"github.com/microsoft/wmi/server2019/root/microsoft/windows/storage"
```

Comment:
this relies on specific server version, does it work on server 2022 and 2025?

Reply:
I always tested on Windows Server 22H2 with this API.
I don't think it would break as long as the new feature introduced in newer API is not used.